### DISCUSSION TITLE
Is there a way to load a plugin to MarkdownIt through conf.py?

### DISCUSSION THREAD

**Discussion by eyllanesc** [executablebooks/MyST-Parser#515] [Q&A]
I see that the MarkdownIt parser loads a predefined list of plugins(colon_fence_plugin, amsmath_plugin, etc) based on the configuration in sphinx(conf.py), but sometimes you want to add custom plugins to MarkdownIt so my question is: How can I do it? since I don't see how to access that object.

**Comment by adam-grant-hendry** [comment]
@eyllanesc See [Issue #632](https://github.com/executablebooks/MyST-Parser/issues/632)
